bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -431,3 +431,63 @@ spec:
         operator: "Exists"
       - key: "node.kubernetes.io/network-unavailable"
         operator: "Exists"
+---
+
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ovnkube-master-metrics
+  namespace: openshift-ovn-kubernetes
+  annotations:
+    kubernetes.io/description: |
+      RBAC Proxy to expose metrics over HTTPS
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  replicas: 1
```

Comment:
`replicas` has no effect for DaemonSets

Reply:
Oh, copy and paste side effect, removed it from the other one too.